nomenclature/code.py

```diff
@@ -209,3 +209,16 @@ class RegionCode(Code):
     """
 
     hierarchy: str = None
+
+
+class MetaCode(Code):
+    """A subclass of Code
```

Comment:
Maybe writing on the purpose would be better. The fact that this is a subclass of Code is already given by the line above.